README.md

```diff
@@ -1,9 +1,52 @@
-# Northcoders News API
+# 📰 Northcoders News API
 
-We'll have two databases in this project: one for real-looking dev data, and another for simpler test data.
+Hosted: https://nc-news-im.onrender.com/api
+
+# 🤖 About
+
+The nc-news API is a PostgreSQL database designed to access application data programmatically that mimics a real world backend service. It will provide the information to the front-end architecture. The server is able to:
+
+- Respond with a list of available endpoints
+- Respond with a list of topics, articles and users
+- Respond with a single article or article comments based on id
+- Allows articles to be filtered and sorted
+- Add and delete comments
+- Update an article by id
+
+The server has been built using Test-Driven-Development to ensure software stability.
+
+# 📝 How to Build
+
+Cloning repo: In the command line where you would like the repository to sit in your file structure use:
+
+git clone https://github.com/indymoorcroft/back-end-project.git
+
+Dependencies to install:
+
+- dotenv
+- express
+- pg
+- pg-format
+- supertest
+- jest
+- jest-sorted
```

Comment:
just running `npm install` will automatically install all of the dependencies listed in the package.json, so you don't have to list these out, just let them know to run npm install